lib/Service/EndpointService.php

```diff
@@ -500,11 +541,17 @@ private function getObjects(
     ): Entity|array
     {
         if (isset($pathParams['id']) === true && $pathParams['id'] === end($pathParams)) {
-            return $this->replaceInternalReferences(mapper: $mapper, serializedObject: $this->objectService->getOpenRegisters()->renderEntity(
-                entity: $mapper->find($pathParams['id'], extend: $parameters['extend'] ?? $parameters['_extend'] ?? null)->jsonSerialize(),
-                extend: $parameters['_extend'] ?? $parameters['extend'] ?? null),
+
+            $result = $this->replaceInternalReferences(
+				mapper: $mapper,
+				serializedObject: $this->objectService->getOpenRegisters()->renderEntity(
```

Comment:
I rather meant the serializedObject argument here to be set on a line of its own